docs/docs-content/enterprise-version/upgrade/upgrade-vmware/airgap.md

```diff
@@ -0,0 +1,186 @@
+Palette upgrade. Additionally, if the new Palette version updates Kubernetes, you need to apply the Kubernetes updates
+to your Enterprise Cluster profile.
+
+:::
+
+## Prerequisites
+
+- Access to the Palette system console.
+
+- Access to the Palette airgap support Virtual Machine (VM) that you used for the initial Palette installation.
+
+- Refer to [Access Palette](/enterprise-version/#access-palette) to download the following from our support:
+  - The new airgap Palette installation bin.
+  - The OVA with the Operating System (OS) and Kubernetes distribution required for the Palette nodes (if the new
+    version of Palette requires a new version of the OS and Kubernetes).
+
+## Upgrade
+
+1.  Use the following command template to SSH into the Palette airgap support VM. Enter the path to your private SSH
+    key, your username, and the IP or domain of the airgap support VM. The default username is `ubuntu`.
+
+    ```shell
+    ssh -identity_file </path/to/private/key> <username>@<vm-ip-or-domain>
+    ```
+
+    Consider the following command example for reference.
+
+    ```shell
+    ssh -identity_file /docs/ssh-private-key.pem ubuntu@palette.example.com
+    ```
+
+2.  Use the following command to switch to the `root` user account, which you need to proceed with the upgrade.
+
+    ```shell
+    sudo --login
+    ```
+
+3.  Use the following command template to download the new Palette airgap installation bin. Enter the username,
+    password, and the Palette airgap installation URL you received from our support team. In the output file name,
+    replace `<version>` with the Palette version you're downloading.
+
+    ```shell
+    curl --user <username>:<password> --output airgap-<version>.bin <link-to-airgap-installation-bin>
+    ```
+
+    Consider the following command example for reference.
+
+    ```shell
+    curl --user <username>:<password> --output airgap-4.1.12.bin https://software-private.spectrocloud.com/airgap/4.1.12/airgap-v4.1.12.bin
+    ```
+
+4.  Refer to the [Additional Packs](../../install-palette/airgap/supplemental-packs.md) page and update the packages you
+    are currently using. You must update each package separately.
+
+    To update a package, use the following command template to download and execute the pack binary.
+
+    ```shell
+    curl --remote-name <pack-name-url> && chmod +x <pack-name-version>.bin && ./<pack-name-version>.bin
+    ```
+
+    Consider the following example for reference.
+
+    ```shell
+    curl --remote-name https://software-private.spectrocloud.com/airgap/packs/airgap-pack-aws-alb-2.5.1.bin \
+      && chmod +x airgap-pack-aws-alb-2.5.1.bin \
+      && ./airgap-pack-aws-alb-2.5.1.bin
+    ```
+
+    ```shell
+    Verifying archive integrity...  100%   MD5 checksums are OK. All good.
+    Uncompressing Airgap Pack - aws-alb Version 4.0.17  100%
+    Setting up Packs
+    - Pushing Pack aws-alb:2.5.1
+    Setting up Images
+    Setup Completed
+    ```
```

Comment:
Do we need to repeat the information in the additional packs page? I think if we mention the additional packs, that's plenty.